### ISSUE TITLE
Release packages for macOS

### ISSUE THREAD

**Issue by tbcrawford-old** [mattanger/ckb-next#10]
I cannot do this I do not believe even through a PR, but could we get a `ckb beta 0.2.7` pkg release file for easy download and installation for users being directed here? I apologize if this is in the works and you guys are waiting on a few other items before doing a 0.2.7 release but I thought it might be nice to have. Of course, feel free to close this if it is in the works.

**Comment by tatokis** [comment]
We are mostly waiting for the rebranding to be finished, and make sure 
everything is stable.
Once that is done, we are going to provide official binaries.

Until then, you can possibly link your binary to people asking for help.

**Comment by tbcrawford-old** [comment]
That's fine. I wasn't sure what the exact plans were. Thank you

**Comment by ghost** [comment]
The package will be released today.

**Comment by chicks-net** [comment]
Congratulations on your first release!
